torch/csrc/generic/StorageMethods.cpp

```diff
@@ -227,16 +235,30 @@ static PyObject *THPStorage_(setFromFile)(THPStorage *self, PyObject *args)
 {
   HANDLE_TH_ERRORS
   PyObject *file = PyTuple_GET_ITEM(args, 0);
-  int fd = PyObject_AsFileDescriptor(file);
-
   PyObject *offset = PyTuple_GET_ITEM(args, 1);
+  bool is_real_file = PyTuple_GET_ITEM(args, 2) == Py_True;
+
+  if (!is_real_file) {
+    // offset can be implemented with a call to the Python object's seek(),
+    // but it's not necessary to do reading at this stage.
```

Comment:
This comment is a bit confusing. Yes, it's clear we don't read here, but we would in a function that's one line below, right?